temporalcli/commands.taskqueue.go

```diff
@@ -10,31 +10,243 @@ import (
 	"go.temporal.io/api/enums/v1"
 	"go.temporal.io/api/taskqueue/v1"
 	"go.temporal.io/api/workflowservice/v1"
+	"go.temporal.io/sdk/client"
 	"go.temporal.io/server/common/tqid"
 )
 
+const taskQueueUnversioned = "UNVERSIONED"
+
+type reachabilityRowType struct {
+	BuildID      string `json:"buildID"`
```

Comment:
```suggestion
	BuildID      string `json:"buildId"`
```

Pedantic, but I think the `Id` form is better JSON

Reply:
Done